server/controllers/bookingRequest.js

```diff
@@ -0,0 +1,121 @@
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+const BookingRequest = require("../models/BookingRequest");
+
+// @route POST /bookingrequests/create
+// @desc Create a new request
+// @access Private
+
+exports.createBookingRequest = asyncHandler(async (req, res, next) => {
+  try {
```

Comment:
We don't need a try/catch block as the asyncHandler will handle promise rejections and forward it to the error handling middleware

Reply:
resolved by 328a3a41cd26e564fc35a38fd11263a1f980079d